site/monitoring.md

```diff
@@ -30,6 +30,7 @@ Monitoring is a broad topic. This guide covers several :
 
  * [What is monitoring](#fundamentals), why it is important, what common approaches to it exist
  * Available monitoring options:
+     * [New Relic RabbitMQ](#with-newrelic) offers multiple dashboards for monitoring nodes
```

Comment:
Sorry but listing New Relic here is clearly overstepping the line. We are listing
a number of general approaches (or available formats, if you will) to monitoring, and listing one vendor only makes sense to the marketing department of that vendor.